### ISSUE TITLE
`update_focused_file` only updates on write

### ISSUE THREAD

**Issue by mrjones2014** [kyazdani42/nvim-tree.lua#839]
**OS**
MacOS 12.0.1

**Neovim version**
```
NVIM v0.6.0
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by brew@HMBRW-A-001-M1-004.local

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/opt/homebrew/Cellar/neovim/0.6.0/share/nvim"

Run :checkhealth for more info
```

**Nvim-tree version**
2e33b16

**Describe the bug**
With the following config:

```
require('nvim-tree').setup({
  update_focused_file = {
    enable = true,
    update_cwd = false,
  }
})
```

Opening a file should reveal the file in `nvim-tree` if the `nvim-tree` window is open. It doesn't. It does, however, reveal the file in `nvim-tree` if I run `:w` on the file, even if there are no changes.

**To Reproduce**
`nvim-tree` config:
```
require('nvim-tree').setup({
  update_focused_file = {
    enable = true,
    update_cwd = false,
  }
})
```

Open a file via `:e file-name` or a fuzzy finder or anything that isn't opening the file through `nvim-tree`, in a folder that is not already expanded in `nvim-tree`.

**Expected behavior**
`nvim-tree` should reveal the file, but it doesn't until you `:w` on the file.

**Comment by kyazdani42** [comment]
i think this behavior was fixed in #836, tell me if it's not, i'll reopen.

**Comment by mrjones2014** [comment]
yep it's fixed, thanks!
